### PR TITLE
HDDS-8630. Document relationship of namespace quota and FS Trash

### PR DESCRIPTION
**Pull request by sumitagrawl**
## What changes were proposed in this pull request?

Documentation related to trash behavior for quota usages

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-8630

## How was this patch tested?

Documentation

### CODE REVIEW

**Review by krishnaasawa1 — APPROVED** [review]
Looks good English version.

File: hadoop-hdds/docs/content/feature/Quota.zh.md [hunk 1]
```diff
@@ -61,6 +61,30 @@ menu:
...
 
+g. 对于移动到垃圾桶的文件和目录，垃圾桶将在以下情况下占用额外的命名空间：
+- 对于桶中垃圾的内部目录路径，即 `/.trash/<user>/<current or timestamp>`
+- 用于将文件/目录移动到某个层次结构中存在的垃圾时创建的额外路径。例如：
+
+```
+- 来源：/<vol>/<bucket>/dir1/dir2/file.txt
+场景 1：
+- 将 file.txt 移至回收站（删除操作时）
+- 使用“dir1 和 dir2”作为额外命名空间创建的垃圾桶与垃圾桶中的源具有相同的路径：
```
Comment (ChenSammi) [inline hadoop-hdds/docs/content/feature/Quota.zh.md:72]: 使用“dir1 和 dir2”作为额外命名空间创建的垃圾桶与垃圾桶中的源具有相同的路径 
->
使用“dir1”和“dir2” 在垃圾桶中创建与来源相同的路径

File: hadoop-hdds/docs/content/feature/Quota.md [hunk 2]
```diff
@@ -63,6 +63,31 @@ e. Linked bucket will consume namespace quota of volume.
 
 f. If the cluster is upgraded from old version less than 1.1.0, use of quota on older volumes and buckets(We can confirm by looking at the info for the volume or bucket, and if the quota value is -2 then volume or bucket is old) is not recommended. Since the old key is not counted to the bucket's namespace quota, the quota setting is inaccurate at this point.
 
+g. For files and directory moving to trash, trash will consume extra namespace for below cases:
```
Comment (ChenSammi) [inline hadoop-hdds/docs/content/feature/Quota.md:66]: I would suggest adding “For FSO bucket” at the beginning of the sentence.

Comment (sumitagrawl) [reply hadoop-hdds/docs/content/feature/Quota.md]: @ChenSammi changes updated.

**Review by ChenSammi — APPROVED** [review]

**Pull request merged**